cartridges/int_cloudinary/cartridge/js/cloudinaryWidgets.js

```diff
@@ -5,17 +5,23 @@ var $ = require('jquery');
 window.renderCloudinaryGalleryWidget = function () {
     var imgUrls;
     var $cldEl = $('.cloudinary-data-container');
-    var cloudinaryObj = $cldEl.data('cloudinary');
 
-    if (cloudinaryObj) {
-        if (cloudinaryObj.galleryEnabled && typeof cloudinary !== 'undefined') {
-            var galleryOptions = cloudinaryObj.images.galleryWidget.options;
-            window.cldGallery = cloudinary.galleryWidget(galleryOptions); // eslint-disable-line no-undef
-            cldGallery.render(); // eslint-disable-line no-undef
-        } else if (cloudinaryObj.images && cloudinaryObj.images.imageURLs) {
-            imgUrls = cloudinaryObj.images.imageURLs;
+    $cldEl.each(function() {
+        var cloudinaryObj = $(this).data('cloudinary');
+
+        if (cloudinaryObj) {
+            if (cloudinaryObj.galleryEnabled && typeof cloudinary !== 'undefined') {
+                var galleryOptions = cloudinaryObj.images.galleryWidget.options;
+                if (cloudinaryObj.domain !== 'res.cloudinary.com') {
+                    galleryOptions.SecureDistribution = cloudinaryObj.domain;
```

Comment:
per testing, please add `galleryOptions = true;` here and in the other galleryOptions changes

Reply:
This is the correct syntax (please change in all relevant places)
`galleryOptions.privateCdn = true;`